src/modules/project/pages/projectView/views/projectCreatorViews/sections/rewards/ProjectManageRewards.tsx

```diff
@@ -14,7 +16,7 @@ export const ProjectManageRewards = () => {
       px={{ base: '10px', lg: '40px' }}
       spacing={{ base: '10px', lg: '20px' }}
     >
-      <CardLayout h="auto" padding="30px 30px" minWidth="100%">
+      <CardLayout h="auto" padding={isMobile ? '10px 10px' : '20px 20px'} minWidth="100%" noMobileBorder>
```

Comment:
I've started using the `standardPadding` constant for this, as it is used so much.

Reply:
Nice catch this is super useful!